test/integrations/destinations/slack/common.ts

```diff
@@ -0,0 +1,166 @@
+import { generateMetadata, overrideDestination } from '../../testUtils';
```

Comment:
Lets try to see if we can build make common utils for these in `testUtils.ts` I think there are some functions which are overllaping if those are not use we can replace more generics like you have built for the slack

Reply:
right now I don't see much scope but we can take look at that later.